src/Raven.Server/Documents/Handlers/Admin/RachisAdminHandler.cs

```diff
@@ -96,6 +96,8 @@ public Task GetClusterTopology()
             var serverUrl = GetStringQueryString("url");
             ServerStore.EnsureNotPassive();
             await ServerStore.AddNodeToClusterAsync(serverUrl);
+            // wait for the node to become a voting member
+            await ServerStore.WaitForNodeBecomeMember(serverUrl);
```

Comment:
Not sure that this is a good idea, on large systems, this can take a lot of time.